libraries/botbuilder-adapter-twilio-whatsapp/src/twilioWhatsAppAdapter.ts

```diff
@@ -57,24 +94,51 @@ export class TwilioWhatsAppAdapter extends BotAdapter {
     protected readonly client: Twilio.Twilio;
     protected readonly channel: string = 'whatsapp';
 
+    // OAuth Properties
+    protected readonly credentials: AppCredentials;
+    protected readonly credentialsProvider: SimpleCredentialProvider;
     /**
      * Creates a new TwilioWhatsAppAdapter instance.
      * @param settings configuration settings for the adapter.
      */
-    public constructor(settings: TwilioWhatsAppAdapterSettings) {
+    public constructor(settings?: Partial<TwilioWhatsAppAdapterSettings>) {
         super();
 
-        this.settings = settings;
+        this.settings = {
+            appId: "",
```

Comment:
Why set them to empty string variables? Shouldn't we do a check here to see if the fields are populated and create the `MicrosoftAppCredentials` and `SimpleCredentialProvider` if populated.